### ISSUE TITLE
Upload and Process button is almost invisible

### ISSUE THREAD

**Issue by porosnie** [eurocontrol-swim/aixm-graph#11]
On my Mac, the " Upload and Process" button is white on light grey. Very difficult to view. Maybe related to the Dark Mode of my Mac?

**Comment by antavelos** [comment]
Could you please upload a screenshot?

**Comment by porosnie** [comment]
Here it is:

<img width="708" alt="Screenshot 2020-05-25 at 12 54 35" src="https://user-images.githubusercontent.com/8693439/82807083-632eb380-9e87-11ea-8008-b62f5ddb3389.png">

**Comment by antavelos** [comment]
Yeah, that's weird. The color should be green like in the DATASET button and since that one is indeed green I don't think it's related to the dark mode. Which browser are you using?
